README.md

```diff
@@ -33,8 +33,11 @@ To deploy the cluster you can use :
     cat inventory/mycluster/group_vars/all/all.yml
     cat inventory/mycluster/group_vars/k8s-cluster/k8s-cluster.yml
 
-    # Deploy Kubespray with Ansible Playbook
-    ansible-playbook -i inventory/mycluster/hosts.ini cluster.yml
+    # Deploy Kubespray with Ansible Playbook - run the playbook as root
+    # The options `-b` is required, for write SSL keys in /etc/ installing packages
```

Comment:
The options `-b` is required, for write SSL keys in /etc/ installing packages

should be:

The options `-b` is required, as for example writing SSL keys in `/etc/`, installing packages